### PR TITLE
Add remaining form tests

### PR DESCRIPTION
**Pull request by eminnett**
The form tests were broken down into multiple chunks. The address form tests have already been written. This PR applies the same pattern to exercise the remaining forms.

### CODE REVIEW

File: spec/forms/waste_exemptions_engine/operator_name_form.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
```
Comment (irisfaraway) [inline spec/forms/waste_exemptions_engine/operator_name_form.rb]: This filename is missing `_spec` at the end, so RSpec isn't running it and it gets missed in the code coverage.

Comment (irisfaraway) [reply spec/forms/waste_exemptions_engine/operator_name_form.rb]: This is actually the case for `operator_postcode_form.rb` and `site_postcode_form.rb` - not from this PR but I just noticed they're missing the `_spec` as well. I think that should bump the coverage up!

File: spec/forms/waste_exemptions_engine/location_form_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+module WasteExemptionsEngine
+  RSpec.describe LocationForm, type: :model do
+    subject(:form) { build(:location_form) }
+
+    LOCATIONS = %w[
+      england
+      northern_ireland
+      scotland
+      wales
+    ].freeze
```
Comment (irisfaraway) [inline spec/forms/waste_exemptions_engine/location_form_spec.rb]: Just occurred to me – if we have `frozen_string_literal: true` at the top, do we need to be explicitly freezing `LOCATIONS` and others like it? I don't think it hurts, but also my understanding is that the comment makes it unnecessary.

Comment (eminnett) [reply spec/forms/waste_exemptions_engine/location_form_spec.rb]: I think Rubocop is doing this. I'll quickly check.

File: spec/forms/waste_exemptions_engine/operator_name_form.rb (hunk 1 above)
Comment (eminnett) [reply spec/forms/waste_exemptions_engine/operator_name_form.rb]: Thanks for noticing. I'll fix this in a moment.

Comment (eminnett) [reply spec/forms/waste_exemptions_engine/operator_name_form.rb]: This is fixed in c0b9b1c.

Comment (irisfaraway) [reply spec/forms/waste_exemptions_engine/operator_name_form.rb]: I noticed because I've done exactly the same thing!

**Pull request merged**